integration/tests/iam_v2p1_fresh_install.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 #shellcheck disable=SC2034
-test_name="iam v2.1 only"
+test_name="iam v2.1 fresh install"
```

Comment:
since fresh `install` means something else in the deploy pipeline, we could replace it with `no legacy policies`